frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -114,15 +114,12 @@ function resetVerifiedUserIdentity() {
 
 function resetForestClientNumberData() {
     formData.value['forestClientNumber'] = '';
+    forestClientData.value = [];
     forestClientNumberVerifyErrors.value = [];
 }
 
-function resetForm() {
-    formData.value = defaultFormData;
-}
-
 function cancelForm() {
-    resetForm();
```

Comment:
we don't use resetForm method anywhere else, removed it and add the reset below.